packages/flutter/lib/src/services/autofill.dart

```diff
@@ -799,13 +799,36 @@ mixin AutofillScopeMixin implements AutofillScope {
       !autofillClients.any((AutofillClient client) => client.textInputConfiguration.autofillConfiguration == null),
       'Every client in AutofillScope.autofillClients must enable autofill',
     );
-    return TextInput.attach(
-      trigger,
-      _AutofillScopeTextInputConfiguration(
-        allConfigurations: autofillClients
-          .map((AutofillClient client) => client.textInputConfiguration),
-        currentClientConfiguration: configuration,
-      ),
+
+    TextInputConfiguration inputConfiguration;
```

Comment:
Why do we need to do this filtering on the framework side? Can the iOS embedder just pick the fields that it finds relevant?

Reply:
I think by doing this we would be able to reduce the size of the messages we have to send to the platform every time the focus changes.